src/App.tsx

```diff
@@ -1,27 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+import { Good } from './types/Good';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const loadAll = () => {
+    getAll()
+      .then((data: Good[]) => (
+        setGoods(data)
+      ));
```

Comment:
```suggestion
    getAll().then(setGoods);
```
There's a shorthand. Use it everywhere where it's possible.